v3/app/models/user.rb

```diff
@@ -65,6 +65,7 @@ def self.sign_out(options)
 
     # DOES USER HAVE PERMISSIONS FOR <ROLE_ID>
     def is_role?(role_id)
+      role_id = role_id.to_s.to_i # ensure it is an integer
       if role_ids.index(".#{Role.role_ids.key("retired")}.")
```

Comment:
In my original implementation "role_id" was <either> false <or> a role_id integer. (I since refactored it to <either> 0 <or> a role_id (integer). When "false" was passed the only way to turn it to an integer was to use ".to_s.to_i" since "false.to_i" returns undefined.  I can pull this out but I figured it does not hurt to leave it in.

re: "retired" and "admin" fields.  I had talked about pulling them into fields and then opted against it.  I think you saw the thread in slack.  Per your comment in slack, "yeah, the multitenant point is a good one. always good to avoid having to do extra coding!" (where I mentioned we did not want to use separate fields for "admin" or "retired" and I would just leave them as implemented, and I discussed extensibility for multi-tenant application when we get there.)

Reply:
DONE.  I added 2 guards instead of using nested if statements.